website/docs/r/resource_instance.html.markdown

```diff
@@ -35,6 +35,33 @@ resource "ibm_resource_instance" "resource_instance" {
 }
 ```
 
```

Comment:
Can we have a title for this section in Bold something 
Provisioning a Hyper Protect DBaaS PostgreSQL Service...
something like this example 
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/website/docs/r/iam_access_group_policy.html.markdown#access-group-policy-for-all-identity-and-access-enabled-services

Reply:
@hkantare I added a title. Please review.